citi-onboarding/valepcd
# Feature/contact

**What I did:**

- Created the contact section and its auxiliary components
- Created a script to send emails

**How to test:**

- First run the server and then the front. Check the screen's responsivity by decreasing the aspect ratio.
Then fill out the forms and check the Vale PCD email box.
